pkg/executor/containerexecutor/templates/job.tmpl

```diff
@@ -9,6 +9,7 @@ spec:
   {{- end }}
   template:
     spec:
+
```

Comment:
we should update helm chart

Reply:
https://linear.app/kubeshop/issue/TKC-991/[infra]-update-executorsjson-file-with-logs-sidecar-build-details added task for that.